pinot-controller/src/main/java/org/apache/pinot/controller/helix/core/minion/PinotHelixTaskResourceManager.java

```diff
@@ -317,8 +317,10 @@ public synchronized String submitTask(String parentTaskName, List<PinotTaskConfi
   public synchronized Set<String> getTasks(String taskType) {
```

Comment:
Annotate it with `@Nullable`, also add some javadoc explaining when will it return null